src/components/ec-letter-icon/ec-letter-icon.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div
+    data-test="ec-letter-icon"
+    class="ec-letter-icon"
+    :class="{ 'ec-letter-icon--clickable': isClickable }"
+  >
+    <svg
+      data-test="ec-letter-icon__svg"
+      class="ec-letter-icon__svg"
+      :width="size"
+      :height="size"
+    >
+      <text
+        x="50%"
+        y="50%"
+        dominant-baseline="central"
+        text-anchor="middle"
+        fill="currentColor"
+        data-test="ec-letter-icon__svg__text"
+        class="ec-letter-icon__svg__text"
+        :font-size="fontSize"
+      >
+        {{ firstLetter }}
+      </text>
+    </svg>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { computed } from 'vue';
+
+import type { LetterIconProps } from './types';
+
+const props = withDefaults(defineProps<LetterIconProps>(), {
+  size: 24,
+  isClickable: false,
+});
+
+const firstLetter = computed(() => props.text[0]);
+const fontSize = computed(() => props.size / 2);
```

Comment:
just a question: why do we have to divide it? Can't we set the default to 12 and send the correct prop?